Add Pontem wallet plugin

lusd opened the pull request:
Hello ^^
We published Pontem plugin to npm:
https://www.npmjs.com/package/@pontem/wallet-adapter-plugin

lusd committed:
feat(readme): add Pontem wallet plugin

0xmaayan commented:
Hi @lusd ! thanks for publishing it! 
We fixed an issue yesterday with `@aptos-labs/wallet-adapter-core` that caused nextjs apps build to fail, seems like this package uses the old version, could you update your fork with the latest changes on https://github.com/aptos-labs/wallet-adapter-plugin-template/blob/main/package.json#L20 and publish again?
Thanks!

lusd commented:
> Hi @lusd ! thanks for publishing it! We fixed an issue yesterday with `@aptos-labe/wallet-adapter-core` that caused nextjs apps build to fail, seems like this package uses the old version, could you update your fork with the latest changes on https://github.com/aptos-labs/wallet-adapter-plugin-template/blob/main/package.json#L20 and publish again? Thanks!

Hey!
Thanks for feedback. I just updated our plugin with new version of  `@aptos-labe/wallet-adapter-core` and published it to npm. So, our plugin is ready to go.
